.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Title: Provide a general summary of your changes in the Title above -->
+<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao -->
 
 ## Description
 
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail, including a list of changes -->
 
 ## Related Issue
 
 <!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If suggesting a new feature or change, please discuss it in #spectrum_tokens_talk or design workshop, first -->
 <!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
-
-## Motivation and Context
-
-<!--- Why is this change required? What problem does it solve? -->
-
-## How Has This Been Tested?
-
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
-
-## Screenshots (if appropriate):
```

Comment:
I don't feel there will typically be screenshots that make sense to include here. If encountering any problems, designers should reach out in #spectrum_tokens_talk, or add a comment to the commit or PR.

Reply:
I agree, but also appreciate that we are not removing this section. I think adding screenshots when appropriate could be helpful!

Reply:
@nabuhasan @lynnhao  it looks like it is removed currently in the pr.